contrib/python/cmd/remote_client.py

```diff
@@ -0,0 +1,136 @@
+import os
+import getpass
+import argparse
+import images
+import ps, rm, rmi
+import sys
+from utils import write_err
+import pytoml
+
+default_conf_path = "/etc/containers/podman_client.conf"
+
+class HelpByDefaultArgumentParser(argparse.ArgumentParser):
+
+    def error(self, message):
+        write_err('%s: %s' % (self.prog, message))
+        write_err("Try '%s --help' for more information." % self.prog)
+        sys.exit(2)
+
+    def print_usage(self, message="too few arguments"): # pylint: disable=arguments-differ
+        self.prog = " ".join(sys.argv)
+        self.error(message)
+
+
+def create_parser(help_text):
+    parser = HelpByDefaultArgumentParser(description=help_text)
+    parser.add_argument('-v', '--version', action='version', version="0.0",
+                        help=("show rpodman version and exit"))
+    parser.add_argument('--debug', default=False, action='store_true',
+                        help=("show debug messages"))
+    parser.add_argument('--run_dir', dest="run_dir",
+                        help=("directory to place socket bindings"))
+    parser.add_argument('--user', dest="user",
+                        help=("remote user"))
+    parser.add_argument('--host', dest="host",
+                        help=("remote host"))
+    parser.add_argument('--remote_socket_path', dest="remote_socket_path",
+                        help=("remote socket path"))
+    parser.add_argument('--identity_file', dest="identity_file",
+                        help=("path to identity file"))
+    subparser = parser.add_subparsers(help=("commands"))
+    images.cli(subparser)
```

Comment:
If we push these into a subdirectory then we can just iterate over them rather than hard coding a list here.